index.html

```diff
@@ -13,28 +13,45 @@ <h1>Frontend Dev Quiz</h1>
     </header>
     <main>
       <section class="question-card">
-        <p class="question-text">What property flips the axes in flexbox?</p>
+        <p class="question-card__p">What property flips the axes in flexbox?</p>
```

Comment:
Maybe it's a matter of taste, but I would prefer more descriptive class names instead of always including the HTML tags in it. 
```suggestion
        <p class="question-card__question">What property flips the axes in flexbox?</p>
```